data/data-pipeline/data_pipeline/tests/score/test_output.py

```diff
@@ -203,3 +212,44 @@ def test_donut_hole_addition_to_score_n(final_score_df):
     assert (
         new_donuts > 0
     ), "FYI: The adjacency index is doing nothing. Consider removing it?"
+
+
+def test_data_sources(
+    final_score_df, hud_housing_df, ejscreen_df, cdc_places_df
+):
+    data_sources = {
```

Comment:
Like that float thing from my last PR, I'd likely never do this in production code, but in a test I am like "wheee"